### ISSUE TITLE
Can't Save Changes. "Profile Picture could not be mirrored" Error

### ISSUE THREAD

**Issue by mrjarbenne** [hwdsb/doppelme-avatars#18]
When attempting to save changes this error is displayed on the users profile page: 

<img width="779" alt="2022-02-15_10-49-39" src="https://user-images.githubusercontent.com/480667/154099358-d918eb42-fbba-4fa4-b080-1e0866efd4c7.png">

**Comment by r-a-y** [comment]
I've fixed this issue and also fixed the Doppelme avatar so it renders as a square.

There was also a change to the Doppelme API in the last couple of years that cropped the full avatar to 75 x 150. I've fixed this so the full avatar render as a square again, but the avatar size has increased to 200 x 200 from the previous 150 x 150. So older users with 75 x 150 should resave their Doppelme avatar to get the correct full-size avatar again.
